.gitignore

```diff
@@ -1,21 +1,38 @@
 # git ignore patterns
 
+autom4te.cache/
 libltdl/
 
+Makefile.in
+
+aclocal.m4
+compile
+config.guess
+config.sub
+configure
+depcomp
+install-sh
+ltmain.sh
+missing
+
 pynest/pynestkernel.cpp
 
 *.o
 *.lo
 *.la
 .*
 *.log
+*.pyc
+config.status
 Makefile
 build/
 
 /juqueen/
 /extras/nest-config
 /extras/create_release.sh
+/extras/userdoc/cmds/
 /libnestutil/config.h
+/libnestutil/stamp-h1
 /librandom/randomtest
 /libtool
 /nest/nest
```

Comment:
Your change brings back many lines that were removed in #421, probably due to a failed merge. Please remove them again.